res/css/views/rooms/_ReplyPreview.pcss

```diff
@@ -20,6 +20,7 @@ limitations under the License.
     background: $background;
     max-height: 50vh;
     overflow: auto;
+    margin-left: calc(-1 * $spacing-8);
```

Comment:
Instead of introducing a negative margin here (which can be annoying) it may probably be a better option to remove the left padding [here](https://github.com/matrix-org/matrix-react-sdk/pull/9977/files#diff-2573b630037828e2b5b6b7d2838e1c06866752f96be40977576a71999eaacf60R30).